apps/cowswap-frontend/src/legacy/components/TransactionConfirmationModal/DisplayLink.tsx

```diff
@@ -9,12 +11,11 @@ import { ExternalLinkCustom } from './styled'
 type DisplayLinkProps = {
   id: string | undefined
   chainId: number
+  leadToBridgeTab: boolean
   onClick?: Command
```

Comment:
Should we clean up? Doesn't seem `onClick` is used anywhere right now (remove dead code).